app/frontend/src/components/Datepicker.tsx

```diff
@@ -66,6 +66,9 @@ export default function Datepicker({
             onChange={(date) => {
               if (date?.isValid()) onChange(date?.toDate());
             }}
+            openTo="year"
```

Comment:
It doesn't always make sense to open to the year. I think either make this a prop, or get rid of `disableToolbar` so you can select the year that way.

Reply:
Yeah, having a look this morning, I think that it's better just to have the toolbar, but without an `openTo` prop.
If we don't have it then if you misclick the year you can't easily fix it - and we don't have to weirdly scroll from 1900.